src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/RestoreOperation/RestoreUtil.cs

```diff
@@ -0,0 +1,738 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Data.Tools.DataSets;
+using Microsoft.SqlServer.Management.Sdk.Sfc;
+using Microsoft.SqlServer.Management.Smo;
+
+namespace Microsoft.SqlTools.ServiceLayer.DisasterRecovery.RestoreOperation
+{
+    public class RestoreUtil
+    {
+        public RestoreUtil(Server server)
+        {
+            this.server = server;
+            this.excludedDB = new List<string> { "master", "tempdb" };
+        }
+
+        /// <summary>
+        /// Current sql server instance
+        /// </summary>
+        public readonly Server server;
+        private readonly IList<string> excludedDB;
+
+        public List<string> GetTargetDbNamesForPageRestore()
+        {
+            List<string> databaseNames = new List<string>();
+            foreach (Database db in this.server.Databases)
+            {
+                if (!this.excludedDB.Contains(db.Name) &&
+                     (db.Status == DatabaseStatus.Normal || db.Status == DatabaseStatus.Suspect || db.Status == DatabaseStatus.EmergencyMode) &&
+                     db.RecoveryModel == RecoveryModel.Full)
+                {
+                    databaseNames.Add(db.Name);
+                }
+            }
+            return databaseNames;
+        }
+
+        public List<string> GetTargetDbNames()
+        {
+            List<string> databaseNames = new List<string>();
+            foreach (Database db in this.server.Databases)
+            {
+                if (!this.excludedDB.Contains(db.Name))
+                {
+                    databaseNames.Add(db.Name);
+                }
+            }
+            return databaseNames;
+        }
+
+        internal DateTime GetServerCurrentDateTime()
+        {
+            DateTime dt = DateTime.MinValue;
+            /*
+            string query = "SELECT GETDATE()";
```

Comment:
dead code here and below.  If you don't need that code could you please delete..if it's being temporarily disabled could you please add a comment why it's disabled with a date and your alias, so we can keep track of when to reenable